cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/AbstractConnectOperator.java

```diff
@@ -665,26 +673,36 @@ public static void updateStatus(Throwable error, KafkaConnector kafkaConnector2,
         }
     }
 
-    protected class ConnectorStatusAndConditions {
+    protected class ConnectorStatus {
```

Comment:
Since the class consists of `connectorStatus`, `conditions`, and `topics` now, so I considered that the original name seems to be a little bit confusing. Should we keep the original one or use another naming that is less confusing?

```yaml
...
status:
  ...
  conditions:
  - lastTransitionTime: "2021-02-14T16:54:00.124312Z"
    status: "True"
    type: Ready
  connectorStatus:
    connector:
      state: RUNNING
      worker_id: 10.244.0.68:8083
    name: file-stream-source-connector-bak
    tasks:
    - id: 0
      state: RUNNING
      worker_id: 10.244.0.68:8083
    type: source
  topics:
  - example-topic
```